examples/frameworks/cherryapp.py

```diff
@@ -3,8 +3,9 @@
 cherrypy.config.update({'environment': 'embedded'})
 
 class Root(object):
+    @cherrypy.expose
     def index(self):
         return 'Hello World!'
     index.exposed = True
 
-app = cherrypy.Application(Root(), script_name=None, config=None)
+app = cherrypy.Application(Root(), script_name=None, config=None)
```

Comment:
We can probably use `app = cherrypy.tree.mount(Root())` here. Can you please try with the latest CherryPy?